pyat/at/physics/fastring.py

```diff
@@ -142,6 +142,8 @@ def simple_ring(
     name: str = "",
     particle: str | Particle = "relativistic",
     TimeLag: float | Sequence[float] = 0.0,
+    chromx_arr: Sequence[float] = 0.0,
+    chromy_arr: Sequence[float] = 0.0
```

Comment:
Can't you just keep the `Qpx` and `Qpy` existing arguments?
```python
Qpx: float | Sequence[float],
Qpy: float | Sequence[float],
...
chromx_arr = np.ravel(Qpx)  # Forces 1d array
chromy_arr = np.ravel(Qpy)
chrom_maxorder = max(chromx_arr.size,chromy_arr.size)
```
This avoids introducing redundant new arguments.

Reply:
I considered this but Qpx refers to Qprime which is linear chromaticity which may be a bit misleading.

Reply:
Right, but a logic like "ignored if…' is also misleading. Considering that Qpx is the 1st term of the series, that would be preferred solution!